client/rest-high-level/src/main/java/org/elasticsearch/client/Request.java

```diff
@@ -69,8 +71,21 @@ public String toString() {
                 '}';
     }
 
-    static Request ping() {
-        return new Request("HEAD", "/", Collections.emptyMap(), null);
+    // For developers please add your new methods in the alphabetical order
```

Comment:
Agreed. But at least as I was not aware of this convention and did not find it in code comments I added it.
Any suggestion? Remove the comment?